cargo/lib/dependabot/cargo/file_updater/manifest_updater.rb

```diff
@@ -139,7 +139,7 @@ def update_manifest_pin(content:, dep:, old_pin:, new_pin:)
         end
 
         def declaration_regex(dep)
-          /(?:^|["'])#{Regexp.escape(dep.name)}["']?\s*=.*$/i
+          /(?:^|^\s*|^\t*|["'])#{Regexp.escape(dep.name)}["']?\s*=.*$/i
```

Comment:
Does `\s` (any whitespace character) not include `\t` (tab character)?

Reply:
Looks like it should. Will update